### ISSUE TITLE
[QA] Move here button should be enable when we browse to other folders

### ISSUE THREAD

**Issue by davitol** [owncloud/ocis#554]
![moveherelow](https://user-images.githubusercontent.com/10596560/92923499-d872c400-f437-11ea-93f0-0883e50e5dbc.gif)

**Comment by davitol** [comment]
Also we should disable this option in Public Links, due to you access to files with no user account and it should not be possible to move the file

**Comment by jnweiger** [comment]
the gif anim looks like you are in the same folder?

**Comment by davitol** [comment]
> the gif anim looks like you are in the same folder?

Yes, it is the same folder/path. IMHO Move button should be only enable once you move to other path

**Comment by refs** [comment]
I'm also seeing in your gif that the translations are not really working correctly
